source/map.rst

```diff
@@ -0,0 +1,245 @@
+Concepts and Tools for Radiation Shielding Analysis
+====================================================
+
+Over recent years, the set of concepts, methodologies and tools that are
+available for the analysis of radiation transport and effects in shielding
+systems has grown dramatically.  Here we attempt to describe the map of these
+capabilities with mention of specific tools that have gained, or are gaining,
+broader application.
+
+Fundamental Concepts
+--------------------
+
+The fundamental analysis need for any radiation shielding problem is
+**radiation transport**, for which two primary approaches are in widespread
+use: Monte Carlo radiation transport and discrete ordinates radiation
+transport.  Many of the effects of radiation can be calculated directly by the
+tools that implement these capabilities, but determining the induced
+radioactivity caused by neutron irradiation requires **activation analysis**
+to be used in conjunction with radiation transport.
+
+Monte Carlo Radiation Transport
+++++++++++++++++++++++++++++++++
+
+Monte Carlo radiation transport directly simulated the stochastic behavior of
+particles that leave a source and travel around the engineered system.  If
+enough particles are simulated using independent sequences of pseudo-random
+numbers, then their mean behavior can be taken as representative of the real
+system.  The single biggest advantage of Monte Carlo radiation transport is
+its ability to model the entire phase space of radiation transport (space,
+direction and energy) as continuous.  Continuous modeling of space allows for
+arbitrarily complex geometric models with need for approximation.  Taken
+together with continuous treatment of direction, this also means that
+following particles through complex streaming paths is also possible with
+these methods.  Finally, continuous treatment of energy allows less
+approximation of resonance behaviors for particles that are slowing down
+through that energy region.  This continuous modeling capability is the
+primary reason that Monte Carlo radiation transport is widespread for this
+kind of analysis.
+
+The biggest drawback of Monte Carlo techniques is in the modeling of deep
+penetration problems with substantial shielding.  Since such systems are
+designed to prevent most particles from reaching the far side of such shields,
+and Monte Carlo radiation transport essentially simulates the behavior of
+these particles, then very few simulated particles will penetrate the shield.
+This, in turn, results in very low quality results behind thick shields, since
+the quality of statistical results relies on a substantial number of
+contributions.  While it is possible to simply increase the total number of
+simulated particles, a number of schemes have been introduced, collectively
+known as *variance reduction* techniques to help improve this situation.
+
+The most widely used tool for Monte Carlo radiation transport is:
```

Comment:
Hmmm... maybe we should say on neutron-based systems (somehow) rather than just fusion?